src/test/java/hw20.java

```diff
@@ -0,0 +1,26 @@
+import org.example.pages.LoginPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static org.example.WaitUtils.waitUntilVisibilityOfElementLocatedBy;
+
+public class hw20 extends BaseTest {
+
+    @Test
+
+    public void deletePlaylist() throws InterruptedException {
+
+        LoginPage loginPage = new LoginPage(getDriver());
+        loginPage.login("dzmitry.kimber@testpro.io", "1qAY76Rs");
+        WebElement Playlist =  getDriver().findElement(By.cssSelector("[class='playlist playlist']"));
+        WebElement Delete = getDriver().findElement(By.cssSelector("[data-testid='playlist-context-menu-delete-86853']"));
+        Delete.click();
+        waitUntilVisibilityOfElementLocatedBy(getDriver(), By.cssSelector("[data-testid='playlist-context-menu-delete-86853']"));
+
+        Assert.assertTrue(Delete.isDisplayed());
```

Comment:
You got this problem, please correct it.

![image](https://github.com/Dzzmitry/A52-QA-Automation-Framework/assets/104761482/4e2e7ed6-ed78-4042-9726-103b00614143)

HINT

Test error (red). This status is assigned to tests that caused an exception from the tested source code.

The tester needs to identify an issue in their code: Test error. This status is assigned to tests that trigger an exception in the tested source code.
In software testing, the "Test error" status is assigned to tests that trigger an exception in the tested source code. When a test case fails due to an unexpected exception being thrown during its execution, it is marked as a test error. This indicates that the code being tested has not handled the exception properly, and further investigation and debugging are required to address the issue.

This status indicates an error in the code and the tester must fix this error:

![Test error_en](https://github.com/Test-Pro-Bootcamp/A53-QA-Automation-Framework/assets/104761482/701a79a3-7c65-4a84-9180-e221c559ea02)

**NoSuchElementException**  

 Verify the locator: As a first step, you ought to find out what kind of issue you are dealing with. Is the element still there and can't be found, or is it no longer present on the page? To do this, you can setup Selenium to not to close the browser after the test failure, and run the test. Once you get the exception, you can inspect the page that is open in the browser. Double-check that the locator being used is correct and up-to-date. Inspect the HTML source code to confirm that the locator uniquely identifies the target element on the page.